src/cdk/lambda/prismaLayer.ts

```diff
@@ -20,15 +26,61 @@ export interface AppLayerProps extends Partial<LayerVersionProps> {
   bundlePostCommand?: string
 }
 
+// recursively gather hashes of all files in directory
+function dirDigest(directory: string): string[] {
```

Comment:
I wanted to use an `async` version of this but can't because we need to call it in the constructor, which is not `async`